src/main/java/com/adelinasportfolio/emailservice/constants/EmailConstants.java

```diff
@@ -0,0 +1,9 @@
+package com.adelinasportfolio.emailservice.constants;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class EmailConstants {
+    public static final String EMAIL_SENT_SUCCESSFULLY = "Email sent successfully";
```

Comment:
Vad ca folosesti asta doar in Service. Are rost sa tii o clasa intreaga pentru o constanta? Sau te gandesti ca o sa creasca in viitor?

Reply:
Exact.